src/layouts/hass-subpage.ts

```diff
@@ -125,7 +125,14 @@ class HassSubpage extends LitElement {
         .main-title {
           margin: var(--margin-title);
           line-height: 20px;
+          min-width: 0;
           flex-grow: 1;
+          overflow-wrap: break-word;
+          display: -webkit-box;
+          -webkit-line-clamp: 2;
+          -webkit-box-orient: vertical;
+          overflow: hidden;
+          text-overflow: ellipsis;
```

Comment:
```suggestion
          text-overflow: ellipsis;
          padding-bottom: 1px;
```

When you are in 2 lines and have a g it is cut of without this.